main/tmux/APKBUILD

```diff
@@ -30,21 +24,20 @@ build() {
 		--sysconfdir=/etc \
 		--mandir=/usr/share/man \
 		--localstatedir=/var \
-		|| return 1
-	make || return 1
+	make
 }
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" install || return 1
+	make DESTDIR="$pkgdir" install
 
 	install -Dm644 example_tmux.conf \
 		"$pkgdir"/usr/share/doc/$pkgname/examples/$pkgname.conf \
-		|| return 1
+	
 	for file in CHANGES FAQ README TODO; do
 		install -m644 "$file" "$pkgdir"/usr/share/doc/$pkgname/ \
```

Comment:
If you remove the \ here, you can ditch the empty line after too. LGTM otherwise!